starboard/xb1/tools/stop_process_by_id.ps1

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+$ProcessId = $args[0]
```

Comment:
A good chunk of those are entirely useless. We can dump them in, as it's a straight copy, but you may want to just exclude some of those ps1 files that are not referenced by anything.

Fine either way

Reply:
Ah, yeah. Likely should have done more internal cleanup prior, but I'll create a tracking bug to review which of these are needed and come back around. b/290063150